### PR TITLE
Add Trust Wallet security info

### PR DESCRIPTION
**Pull request by danpaul000**
#### Problem
No security information about the risks of using Trust Wallet.

#### Summary of Changes
Add it.

### CODE REVIEW

File: docs/src/wallet-guide/apps.md [hunk 1]
```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
...
+passcode to open the app itself.  Enable Trust Wallet passcode by opening
+the app and going to Settings -> Security -> Passcode.
+
+Additionally, anyone who has access to your seed phrase will be able to recreate
+your Trust Wallet keys on a different device, and sign transactions from that
+device, rather than on your own phone or tablet.
+The seed phrase is displayed when a
+new wallet is created, and it can also be viewed at any later time in the app by
+going to Setting -> Wallets, and under the Options menu for a particular wallet,
+tap "Show Recovery Phrase".  If you use an insecure method of electronic backup,
```
Comment (t-nelson) [inline docs/src/wallet-guide/apps.md:25]: This first sentence feels weird.  I'm not sure if it's the out of place line break, or the two Oxford commas :confused: 

Maybe breaking the settings navigation out to a list would clean things up?

File: docs/src/wallet-guide/apps.md [hunk 2]
```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
 currently the easiest and fastest way to get set up with a new wallet on Solana.
 The app is free and getting your wallet set up only takes a few minutes.
 
+### Trust Wallet Security
+
+Tokens held in Trust Wallet are only as secure as the device on which the app is
+installed.  Anyone who is able to unlock your phone or tablet may be able to
+use the Trust Wallet app and transfer your tokens.  It is *highly* recommended
```
Comment (garious) [inline docs/src/wallet-guide/apps.md:14]: We should avoid making recommendations. Instead, how about something along the lines of,  "To improve security, you can add a passcode to the Trust Wallet application. To add a Trust Wallet passcode, open the app and go to Settings -> Security -> Passcode."

File: docs/src/wallet-guide/apps.md [hunk 3]
```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
...
+that you always use a strong passcode to lock your device and use a different
+passcode to open the app itself.  Enable Trust Wallet passcode by opening
+the app and going to Settings -> Security -> Passcode.
+
+Additionally, anyone who has access to your seed phrase will be able to recreate
```
Comment (garious) [inline docs/src/wallet-guide/apps.md:19]: This is missing a step. It's more like anyone who gains access to the application can access your recovery phrase, and once someone has the recovery phrase, they can recreate your wallet on another device.

File: docs/src/wallet-guide/apps.md [hunk 4]
```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
...
+device, rather than on your own phone or tablet.
+The seed phrase is displayed when a
+new wallet is created, and it can also be viewed at any later time in the app by
+going to Setting -> Wallets, and under the Options menu for a particular wallet,
+tap "Show Recovery Phrase".  If you use an insecure method of electronic backup,
+such as emailing yourself your seed phrase, if the email is intercepted or if
+anyone else has access to the same email account, they would be able to create
+your Trust Wallet keys on another device.
+
+For these reasons, we do not recommend storing large quantities of SOL using
```
Comment (garious) [inline docs/src/wallet-guide/apps.md:30]: I don't know if this paragraph is necessary.  The point of the "Security" sections under each wallet is to allow a newcomer to decide which wallet best fits their needs.  If the text above is clear, this part should be obvious.

File: docs/src/wallet-guide/apps.md [hunk 5]
```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
...
+recovery seed phrase.
+Anyone who has access to your seed phrase will be able to recreate
+your Trust Wallet keys on a different device. From there, they could
+sign transactions from that device rather than on your own phone or tablet.
+The seed phrase is displayed when a new wallet is created and it can also be
+viewed at any later time in the app by following these steps:
+ - Go to Setting -> Wallets
+ - Under the Options menu for a particular wallet tap "Show Recovery Phrase"
+
+If you use an insecure method of electronic backup,
```
Comment (garious) [inline docs/src/wallet-guide/apps.md:29]: Can you move or delete this paragraph? It applies to all wallets.

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

**Pull request merged**